### PR TITLE
exported a function (getXDClueAnswer)

### PR DESCRIPTION
**Pull request by cod1r**
the exported function helps convert a JSON clue to it's XD clue

### CODE REVIEW

**Comment by cod1r** [comment]
this should make it easy in to convert a JSON clue to it's xd clue format in
`PuzzleSubmissionEditableClueTitle.tsx`

```ts
  const fullAnswer = answerWithSplits(clue.answer, clue.splits)
```

to

```ts
 const fullAnswer = getXDClueAnswer({ json: puzzleJSON, clue, splitChar: "|" })
```

but I might be wrong

**Review by samanpwbb — COMMENTED** [review]
We are using this in some place where we are just acting on a single clue somewhere in the studio interface? Makes sense.

**Review by samanpwbb — APPROVED** [review]

File: lib/JSONtoXD.ts [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+ * inserts the rebus symbol where it needs to be, adds the splits in the proper locations, then replaces the
+ * rebus symbol with the word
+ *
+ * @param {Object} getXDClueAnswerProps - has CrosswordJSON, clue and splitChar
+ * @property {CrosswordJSON} getXDClueAnswerObject.json - CrosswordJSON information that includes meta.rebus
+ * @property {Clue} getXDClueAnswerObject.clue - Clue that includes split locations
+ * @property {string} getXDClueAnswerObject.splitChar - split character to insert
+ * @returns {string} clue answer in it's xd format form
+ */
+export function getXDClueAnswer({ json, clue, splitChar }: { json: CrosswordJSON, clue: Clue, splitChar: string }) {
```
Comment (orta) [inline lib/JSONtoXD.ts:23]: Can you make this function act like the rest of this repo please - e.g. inline arguments to functions

File: index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export * from "./lib/types"
 
-export { JSONToXD } from "./lib/JSONtoXD"
+export { JSONToXD, getXDClueAnswer } from "./lib/JSONtoXD"
```
Comment (orta) [inline index.ts:3]: I'm not sure on the naming of this function, what is it actually trying to do? 

- `resolveFullClueAnswer`  maybe?

**Pull request merged**